_ml-commons-plugin/gpu-acceleration.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: default
+title: GPU acceleration
+parent: Model-serving framework
+nav_order: 150
+---
+
+# GPU acceleration 
+
+When running a natural language processing (NLP) model in your OpenSearch cluster with a machine learning (ML) node, you can achieve better performance on the ML node using graphics processing unit (GPU) acceleration. GPUs can work in tandem with the CPU of your cluster to speed up the model upload and training. 
+
+## Supported GPUs
+
+Currently, ML nodes support any CUDA-enabled GPUs, including the following:
```

Comment:
We should clarify here the supported CUDA version is 11.6, supported pytorch version is 1.12.1